### PR TITLE
Upgrade OpenCVSharp to 4.5.0

### PR DESCRIPTION
**Pull request by m-roberts**
First step of #3

### CODE REVIEW

File: libs/build-opencv-sharp.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+mkdir build
+cd build
+cmake ..
+time make -j4
+sudo make install
+sudo ldconfig
+##### Install
+sudo tar -xzvf opencv_opencvsharp_libs.tar.gz -C /usr/local/lib/
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+sudo ldconfig
```
Comment (colombod) [inline libs/build-opencv-sharp.sh]: this is just pushing all stuff to the /usr/local/lib, we do not want to do this, am I right?

**Comment by colombod** [comment]
Is there a docker image based on pi-top os we can use to compile this code?

**Review by colombod — APPROVED** [review]

File: libs/build-opencv-sharp.sh (hunk 1 above)
Comment (m-roberts) [reply libs/build-opencv-sharp.sh]: I think that this is an acceptable first version solution. Let's discuss a better implementation when we are a bit further along!

**Comment by m-roberts** [comment]
> Is there a docker image based on pi-top os we can use to compile this code?

Not yet - let's figure out a more sustainable solution when we have a few more pressing things ironed out

**Pull request merged**